### PR TITLE
Create french Documentation chapter

### PR DESCRIPTION
**Pull request by jeamick**
Start Documentation Chapter

ready for review : @Natim @skade 
Thanks

### CODE REVIEW

File: presentation/chapters/fr-FR/documentation.chapter [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+## `rustdoc`
+
+Rust fournit un outil de documentation standard appelé `rustdoc'. Il est couramment utilisé par l'intermédiaire de `cargo doc`.
+
+De ce fait, le code en Rust est presque toujours documenté dans un format commun.
+---
+
+## Documentation de `std`
+
+La Documentation de la Librarie standard est hebergee au lien suivant :https://doc.rust-lang.org/std/.
```
Comment (Natim) [inline presentation/chapters/fr-FR/documentation.chapter]:
```suggestion
La documentation de la bibliothèque standard est hébergée ici : https://doc.rust-lang.org/std/
```

File: presentation/chapters/fr-FR/documentation.chapter [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+Une version locale, hors ligne, peut être ouverte avec :
+
+<pre><code data-source="chapters/shared/code/documentation/1.bash" data-trim="hljs bash"></code></pre>
+
+---
+
+## Documentation de Crates
+
+La Documentation pour crates est hebergée sur http://crates.io/  peut etre trouvee a l'adresse suivente: https://docs.rs/.
```
Comment (Natim) [inline presentation/chapters/fr-FR/documentation.chapter]:
```suggestion
La documentation pour crates hébergée sur http://crates.io/  peut-être trouvée a l'adresse suivante : https://docs.rs/
```

File: presentation/chapters/fr-FR/documentation.chapter [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+Certaines crates peuvent également avoir d'autres documents trouvés via le lien "Documentation" sur le lien suivant: http://crates.io/.
```
Comment (Natim) [inline presentation/chapters/fr-FR/documentation.chapter]:
```suggestion
La documentation de certaines crates peut-également être accessible avec le lien "Documentation" du site http://crates.io/.
```

File: presentation/chapters/fr-FR/documentation.chapter [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+## Navigation
+
+Les arguments et les types de retour des fonctions sont des liens vers leurs types respectifs.
+La barre latérale de gauche permet de naviguer rapidement vers les autres parties du module.
+
+---
+
+## Integration de Cargo
+
+Cette Commande crée et ouvre les documentations de votre project actuel:
```
Comment (Natim) [inline presentation/chapters/fr-FR/documentation.chapter]:
```suggestion
Cette commande crée et ouvre la documentation de votre projet courant :
```

**Comment by jeamick** [comment]
@skade  any thought on this ?

**Pull request merged**